frontend/src/components/domains/DKIMKeyViewer.vue

```diff
@@ -23,6 +23,10 @@ export default {
         }
       }
       return result.join('\n')
+    },
+    copyDNSToClipboard () {
+      const text = document.getElementById('dkimdns').textContent
```

Comment:
Instead of doing this, you could create a computed property (to generate the content of the pre tag above)